docker_entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
`./docker` is really for supplementary docker images needed by docker-compose so maybe it's a bad name docker_entrypoint could go in `./bin/` if you don't like it toplevel